src/components/TransactionsList.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react'
+import Transaction from './Transaction'
 
-const TransactionsList = () => {
-
+const TransactionsList = (props) => {
+  const transactions = props.transactions.map((tx, i) =>
+    <Transaction transaction={tx} key={i}/>
```

Comment:
double check how you're passing props into this component - remember that the functional component will take in an object.